### PR TITLE
Exchange term 'user' with more descriptive words

### PR DESCRIPTION
**Pull request by mitelg**
Using the term "user" in this guide, let to misunderstandings of the payment feature. In this particular case it was the "message" which could be send if the payment fails. 

I exchanged all occurrences, with "customer" or "merchant"

### CODE REVIEW

**Review by mstegmeyer — APPROVED** [review]

**Review by lernhart — APPROVED** [review]

File: guides/plugins/apps/payment.md [hunk 1]
```diff
@@ -340,11 +359,22 @@ class PaymentController {
 
 ## Prepared payments
 
-With Shopware 6.4.9.0, you can use prepared payments to enhance your checkout process beyond forwarding to a payment provider. This feature enables you to integrate more deeply into the checkout process. This method allows you to prepare the payment before placing the order, e.g., with credit card fields on the checkout confirmation page. Once you add specific parameters to the order placement request in the Storefront, which is also known as the checkout confirmation form, you can pass these parameters to your prepared payment handler. This enables your payment handler to capture the payment successfully when the order is placed.
+With Shopware 6.4.9.0, you can use prepared payments to enhance your checkout process beyond forwarding to a payment provider.
```
Comment (Isengo1989) [inline guides/plugins/apps/payment.md:362]:
```suggestion
With Shopware `6.4.9.0`, you can use prepared payments to enhance your checkout process beyond forwarding to a payment provider.
```

File: guides/plugins/apps/payment.md [hunk 2]
```diff
@@ -620,7 +657,11 @@ function refund(RequestInterface $request): ResponseInterface
 Recurring orders and payments require the Subscriptions feature, available exclusively in our [paid plans](https://www.shopware.com/en/pricing/).
 :::
 
-Recurring payments are a special case of payment that is used for handling recurring orders, such as subscriptions. The request and response payloads are similar to the synchronous payment flow. At this point, a valid running billing agreement between the customer and the PSP should exist. Use any of the other payment flows to capture the initial order and create such an agreement during the checkout. Afterwards, the payment can be captured via this flow for every recurring payment order.
+Recurring payments are a special case of payment that is used for handling recurring orders, such as subscriptions.
+The request and response payloads are similar to the synchronous payment flow.
+At this point, a valid running billing agreement between the customer and the PSP should exist.
+Use any of the other payment flows to capture the initial order and create such an agreement during the checkout.
+Afterwards, the payment can be captured via this flow for every recurring payment order.
```
Comment (Isengo1989) [inline guides/plugins/apps/payment.md:664]:
```suggestion
Afterward, this flow can capture the payment for every recurring payment order.
```

File: guides/plugins/apps/payment.md [hunk 3]
```diff
@@ -543,14 +574,20 @@ class PaymentController {
...
+your customer might not be able to use the prepared payment.
+This is especially valid for after order payments, since there the order already exists.
+For these cases, you should still offer a traditional synchronous / asynchronous payment flow.
+Don't worry, if you have set the transaction state in your capture call to anything but open, the asynchronous payment process will not be started immediately after the prepared payment flow.
 :::
 
 ## Refund
 
-With Shopware 6.4.12.0, we have also added basic functionality to be able to refund payments. Your app will need to register captured amounts and create and persist a refund beforehand for Shopware to be able to process a refund of a capture.
+With Shopware 6.4.12.0, we have also added basic functionality to be able to refund payments.
```
Comment (Isengo1989) [inline guides/plugins/apps/payment.md:586]:
```suggestion
With Shopware `6.4.12.0`, we have also added basic functionality to be able to refund payments.
```

**Review by Isengo1989 — COMMENTED** [review]
Some small adjustments :) Thx for the PR

**Review by Isengo1989 — APPROVED** [review]

**Pull request merged**